server/apps/users/views.py

```diff
@@ -131,3 +132,100 @@ def get(request):
         serializer = TodoSerializer(todos, many=True)
 
         return Response(serializer.data)
+
+    # Update to'do
+    @staticmethod
+    def put(request, **kwargs):
+        user = request.user
+        todo_id = request.data['todo_id']
+
+        # Get the to'do that user want to update from database and check if to'do in our database has current user as a editor
+        todo = Todo.objects.filter(id=todo_id).first()
+        editors = todo.editors.all()
+
+        # If current user is not an owner or editor for the to'do then don't process the request
+        if user != todo.owner and user not in editors:
+            return Response(OPERATION_NOT_ALLOWED)
+
+        # Set current user as updated by
+        request.data['updated_by'] = user.name
+
+        serializer = TodoSerializer(todo, data=request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+
+        return Response(serializer.data)
+
+    # Delete to'do
+    @staticmethod
+    def delete(request, **kwargs):
+        todo_id: int = request.GET['todo-id']
```

Comment:
Don't use static methods here, override the already available delete method from `DeleteAPIView`.

Reply:
Updated